static/css/components/header-bar.less

```diff
@@ -538,6 +538,7 @@
     .logo-component {
       .logo-icon {
         height: 47px;
+        width: 194px;
```

Comment:
The /static/images/openlibrary-logo-tighter.svg image appears to be 189px by 47px yet we are specifying width="194". I think both this value and the image width attribute should be 189px.